src/stream-tcp.c

```diff
@@ -925,7 +926,11 @@ static int StreamTcpPacketStateNone(ThreadVars *tv, Packet *p,
     /* SYN/ACK */
     } else if ((p->tcph->th_flags & (TH_SYN|TH_ACK)) == (TH_SYN|TH_ACK)) {
         if (!stream_config.midstream && stream_config.async_oneside == FALSE)
-            return 0;
+            if (stream_config.midstream_policy == EXCEPTION_POLICY_IGNORE)
```

Comment:
does this mean that `stream.midstream-policy=pass-flow` effectively also adds `stream.midstream=true`?

Reply:
Indeed, with `stream.midstream-policy=pass-flow` it acts as if `stream.midstrem=true`, allowing for the engine to still log the flow etc, but without detection. That was what I understood that should happen 😅